iati/guidance_and_support/templates/guidance_and_support/guidance_group_page.html

```diff
@@ -43,6 +43,7 @@ <h2 class="type-m type-m--trailer heading">{% trans "In this section" %}:</h2>
         {% endif %}
           <h2 class="card__heading"><a href="{% slugurl_trans group.page.slug %}"><span>{{ group.page.heading }}</span></a></h2>
           <p class="card__excerpt">{{ group.page.excerpt|default:"" }}</p>
+          <p class="card__excerpt--truncated">{{ group.page.excerpt|truncatechars:35|default:"" }}</p>
```

Comment:
I'd like to see repeated markup abstracted out to includes in the future. It's not within the scope of what we're doing right now, but it's been filed into the tech debt pile in my brain!